### PR TITLE
SISRP-28019 - Committees card need to show Insider/outsider details for committee members

### PR DESCRIPTION
**Pull request by nkutub**
https://jira.berkeley.edu/browse/SISRP-28019

* added new committee member roles to the backend and new sections to frontend for inside and outside members. 
* used CS link directive for committees link 
* small text changes on UI
* removed filter for student committees to allow for active and completed committees to be rendered.

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/academicsHigherDegreeCommitteesController.js [hunk 1]
```diff
@@ -57,6 +57,14 @@ angular.module('calcentral.controllers').controller('AcademicsHigherDegreeCommit
     return committeeData.committeeMembers.coChair;
   };
 
+  var getCommitteeInsideMembers = function(committeeData) {
+    return committeeData.committeeMembers.insideMembers;
+  };
+
+  var getCommitteeOutsideMembers = function(committeeData) {
+      return committeeData.committeeMembers.outsideMembers;
+    };
```
Comment (raydavis) [inline src/assets/javascripts/angular/controllers/widgets/academicsHigherDegreeCommitteesController.js]: Overenthusiastic indent

File: src/assets/javascripts/angular/factories/higherDegreeCommitteeFactory.js [hunk 2]
```diff
@@ -6,8 +6,8 @@ var angular = require('angular');
  * Higher Degree Committees Factory
  */
 angular.module('calcentral.factories').factory('higherDegreeCommitteeFactory', function(apiService) {
-  // var url = '/dummy/json/committees.json';
-  var url = '/api/my/committees/';
+  var url = '/dummy/json/committees.json';
```
Comment (raydavis) [inline src/assets/javascripts/angular/factories/higherDegreeCommitteeFactory.js]: Oops

**Comment by raydavis** [comment]
Looks like a lot of last minute rethinking happened! Three lines need a little tweak; otherwise, the PR seems good to merge.

Also, would you mind adding a screenshot of the new Inside/Outside layout to the JIRA ticket? CC devs have usually done that as a way to document post-facto in-the-wild design changes, and it's pretty handy for review, QA, etc.

**Comment by nkutub** [comment]
@raydavis , thanks for review. I also added image to the JIRA.

**Comment by raydavis** [comment]
Thanks, @nkutub !

**Pull request merged**